services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/NodeManagerImplAutoWiredTest.java

```diff
@@ -372,7 +373,9 @@ public void testUpdateWithVersion() throws DatastoreException, InvalidModelExcep
 		String valueOnSecondVersion = "Value on the second version.";
 		annosToUpdate.addAnnotation("stringKey", valueOnSecondVersion);
 		// call under test
-		Node afterUpdate = nodeManager.update(adminUserInfo, updatedNode, null,  annosToUpdate, true);
+		Node afterUpdate = nodeManager.update(adminUserInfo, updatedNode, null, true);
+		annosToUpdate.setEtag(afterUpdate.getETag());
+		nodeManager.updateUserAnnotations(adminUserInfo, updatedNode.getId(), annosToUpdate);
 		assertNotNull(afterUpdate);

```

Comment:
set etag and update user annos no longer occurs with the method under test so the test should change accordingly.